Add explanation on how Packit closes bugzillas (#806)
packit/packit.dev: docs/configuration/downstream/pull_from_upstream.md

src.fedoraproject.org when a new upstream release happens using a notification
from [release-monitoring.org](https://release-monitoring.org/). Upstream Release Monitoring creates a Bugzilla
for the new upstream release and this Bugzilla is by default referenced in the dist-git changelog as
`- Resolves rhbz#xz` and `- Resolves rhbz#xz` in the commit message. 

> The `pull_from_upstream` job will close the Bugzillas opened by the Upstream Release Monitoring on your behalf 
> through the [Bodhi updates](https://fedora-infra.github.io/bodhi/6.0/user/automatic_updates.html#associate-bugs-to-automatic-updates). 

Reviewer: Correct me if I'm wrong, but I think this is more accurate:

```suggestion

Bodhi updates created by the `bodhi_update` job as well as [automatic Bodhi updates](https://fedora-infra.github.io/bodhi/6.0/user/automatic_updates.html) will have this Bugzilla assigned and it will be closed when they reach stable.
```

There is also an env variable with the list of bugs to be closed
`PACKIT_RESOLVED_BUGS` that you can use in the case you want to customize the changelog creation but still have
the bugs automatically closed.

This job behaviour can be customized

> using [actions](/docs/configuration/actions#syncing-the-release).

Reviewer 2: I would mention that the env variable is exposed for `commit-message` and `changelog-entry` actions

src.fedoraproject.org when a new upstream release happens using a notification
from [release-monitoring.org](https://release-monitoring.org/). Upstream Release Monitoring creates a Bugzilla
for the new upstream release and this Bugzilla is by default referenced in the dist-git changelog as
`- Resolves rhbz#xz` and `- Resolves rhbz#xz` in the commit message. 

> Bodhi updates created by the `bodhi_update` job as well as [automatic Bodhi updates](https://fedora-infra.github.io/bodhi/6.0/user/automatic_updates.html), for `rawhide`, will have this Bugzilla assigned and it will be closed when they reach stable.

Reviewer: I wouldn't mention rawhide, because (as mentioned in the link) it's not the only branch automatic updates are created for, after branching happens there is a period of time when the branched release also receives automatic updates.

> There is also an env variable with the list of bugs to be closed
> `PACKIT_RESOLVED_BUGS` that you can use in the case you want to customize the changelog creation but still have
> the bugs automatically closed.
> 
> This job behaviour can be customized
> using [actions](/docs/configuration/actions#syncing-the-release). The `PACKIT_RESOLVED_BUGS` env variable is 
> available both for the `commit-message` and `changelog-entry` actions.

Reviewer: I would probably merge the two paragraphs:

```suggestion
This behaviour can be customized
using [actions](/docs/configuration/actions#syncing-the-release). In both `commit-message` and `changelog-entry` actions you can use `PACKIT_RESOLVED_BUGS` environment variable containing the list of Bugzillas to be closed.
```

Author: I am not sure here because the *customizable behaviour* is not related just with closing Bugzillas, but this is about the entire pull from upstream job 🤔.
